image-counter-lambda/src/handler.ts

```diff
@@ -0,0 +1,54 @@
+import getCredentials from "./getCredentials";
+import fetch from "node-fetch";
+import CloudWatch from "aws-sdk/clients/cloudwatch";
+
+interface MediaAPICredentials {
+  baseUrl: string;
+  "X-Gu-Media-Key": string;
+}
+
+const getImageCount = async (
+  credentials: MediaAPICredentials
+): Promise<number> => {
+  const response = await fetch(credentials.baseUrl + "/images", {
```

Comment:
We could use the `length` querystring to control the response size. That is, `length=0` will yield a smaller response than the default which is `length=10`.